localisation/00_zuwc_united_localisation.csv

```diff
@@ -2881,13 +2880,104 @@ is_dragon_race_trigger_tooltip;Is a dragon;;;;;;;;;;;;;x
+EVTOPTA_WCTFT_141;The deed is done, may my own legend begin!;;Die Tat ist vollbracht, m�ge meine eigene Legende beginnen!;;;;;;;;;;;x
+EVTDESC_WCTFT_200;To your surprise, you have lost track of [target_silver_leader.GetDynName]. Your informants told you that [target_silver_leader.GetSheHe] has died, others claim that [target_silver_leader.GetSheHe] has went into hiding after hearing rumours of an assassination attempt on [target_silver_leader.GetHerHis] life... In any case, your little plot turned out to be a failure.;;Zu Eurer �berraschung habt Ihr die Spur von [target_silver_leader.GetDynName] verloren. Eure Informanten haben Euch mitgeteilt, dass [target_silver_leader.GetSheHe] gestorben sei, andere behaupten, dass [target_silver_leader.GetSheHe] untergetaucht sei, nachdem [target_silver_leader.GetSheHe] Ger�chte �ber ein Attentat auf [target_silver_leader.GetHerHis] Leben geh�rt hat... Auf jeden Fall hat sich Euer kleiner Plan als Fehlschlag herausgestellt.;;;;;;;;;;;x
+Get_target_province;[target_province.GetName];;[target_province.GetName];;;;;;;;;;;x
+Get_the_city;the city;;die Stadt;;;;;;;;;;;x
+Get_undead;undead;;Untoten;;;;;;;;;;;x
+Get_demons;demons;;D�monen;;;;;;;;;;;x
+Get_void_beings;void beings;;Leerenwesen;;;;;;;;;;;x
+Get_enemies;enemies;;Feinde;;;;;;;;;;;x
+Get_a_border_city;a border city;;eine Grenzstadt;;;;;;;;;;;x
```

Comment:
line 2933 and 2938 don't work well in translation... but it'll work for now

Reply:
@arithon you confuse me! What am I supposed to do now?

Reply:
nothing, so far... I'll have to think about a way to redo the texts where these parts come into play, but for now, it's ok I think.

Reply:
@arithon sooo I resolve it?